docker/functions.go

```diff
@@ -34,3 +34,24 @@ func GetContainerByName(client *dockerclient.Client, name string) (*dockerclient
 
 	return &containers[0], nil
 }
+
+// GetContainerByID looks up the hosts containers with the specified Id and
+// returns it, or an error.
+func GetContainerByID(client *dockerclient.Client, id string) (*dockerclient.APIContainers, error) {
+	containers, err := client.ListContainers(dockerclient.ListContainersOptions{All: true, Size: false})
```

Comment:
Oh was going to say "yeah i is documented" but.. http://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#list-containers doesn't seem to be up-to-date :sweat_smile:.